src/wui/progress_data_wrapper.cpp

```diff
@@ -0,0 +1,35 @@
+#include "../../../lib/Marlin/Marlin/src/gcode/lcd/M73_PE.h"
```

Comment:
writing the path to header is not nice way, you have to "target link library" marlin library